ufo2fontir/src/toir.rs

```diff
@@ -1,8 +1,16 @@
 use fontir::ir;
-use norad::designspace::{self, DesignSpaceDocument};
+use norad::designspace::{self, DesignSpaceDocument, Dimension};
+use ordered_float::OrderedFloat;
 
 use crate::error::Error;
 
+// TODO we will need the ability to map coordinates and a test font that does. Then no unwrap.
```

Comment:
Maybe we should file an issue for this as it's somewhat important and surprisingly involved given the mapping mechanism?

Reply:
https://github.com/googlefonts/fontmake-rs/issues/22